packages/app/src/app/pages/Dashboard/Header/index.tsx

```diff
@@ -87,28 +87,31 @@ export const Header: React.FC<HeaderProps> = React.memo(
         <SearchInputGroup />
 
         <Stack align="center" gap={2}>
-          <Button
-            type="button"
-            variant="link"
-            onClick={() => {
-              track('Dashboard - Invite members');
 
-              /* Only for teams */
-              if (activeTeam !== personalWorkspaceId) {
-                history.push(`${dashboardUrls.teamInvite()}?from-header=1`);
-              } else {
-                history.push(`${dashboardUrls.createTeam()}?from-header=1`);
-              }
-            }}
-            autoWidth
-          >
-            <PlusIcon css={css({ marginRight: 2, width: '.8em' })} /> Invite
-            members
-          </Button>
+        
+        <Link  
```

Comment:
It's happening! 🎉

A few concerns:
- Should this button only appears for those who are on the beta? (feature flaged);
- If so, can we replace "Go to Projects" to "Join the waitlist"? (I think there's an opportunity here);
- The file hasn't been formatted, is anything wrong with your prettier configuration?